src/components/Navbar.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import Image from "next/image";
+
+import { useState } from "react";
+import Link from "next/link";
+import { Equal, X } from "lucide-react";
+import { Button } from "@/components/ui/button";
+
+export function Navbar() {
+  const [showSidebar, setShowSidebar] = useState<boolean>(false);
+  return (
+    <>
+      <div className="w-full flex flex-row justify-between items-center gap-12">
+        <div className="flex flex-row gap-2 items-center mr-auto">
+          <Image
+            src="/assets/logo/promochator_logo.png"
+            alt="Promochator Logo"
+            className="w-6"
+            width={150}
+            height={150}
+          />
+          <p className="text-lg font-semibold">
+            Promo<span className="text-[#5f7ecd]">CHAT</span>or
+          </p>
+        </div>
+        <div className="hidden lg:flex flex-row gap-12 whitespace-nowrap">
+          <Link href="#how-it-works">Jak to działa?</Link>
+          <Link href="#details">Jak ci pomożemy?</Link>
+          <Link href="#terms-of-use">Regulamin</Link>
+        </div>
+        <Link href="https://solvro.pwr.edu.pl/" className="ml-auto" passHref>
+          <Image
+            src="/assets/logo/solvro_dark.png"
+            alt="Solvro Logo"
+            width={150}
+            height={150}
+            className="hidden lg:flex w-40"
+          />
+        </Link>
+        {/* Mobile icon */}
+        <Button
+          size="icon"
+          variant="transparent"
+          title="Otwórz menu"
+          onClick={() => setShowSidebar(true)}
+          className="lg:hidden"
```

Comment:
I think we need to hide "open sidebar" button after opening side bar as the button "shines" through
I'd suggest to use cn() utility function from src/lib/utils.ts or tailwindMerge()
![image](https://github.com/user-attachments/assets/cff1c7a8-e837-47f3-8f5f-6fe8bfb8f82b)